dev/conductor/core/lib/src/state.dart

```diff
@@ -149,6 +149,9 @@ String phaseInstructions(pb.ConductorState state) {
         return <String>[
           'There are no engine cherrypicks, so issue `conductor next` to continue',
           'to the next step.',
+          '\n',
+          'Note: Create a new entry for this release in go/release-eng-retros',
```

Comment:
I think we should emphasize this, and do something like

```
*****************************************************
* Create a new entry in http://go/release-eng-retros *
*****************************************************
```

Reply:
Love this idea! Will update.